Sources/BraveWallet/Crypto/Transaction Confirmations/TransactionStatusView.swift

```diff
@@ -0,0 +1,181 @@
+              openWalletURL?(url)
+            }
+          } label: {
+            HStack {
+              Text(Strings.Wallet.viewOnBlockExplorer)
+              Image(systemName: "arrow.up.forward.square")
+            }
+            .foregroundColor(Color(.braveBlurpleTint))
+            .font(.subheadline.bold())
+          }
+          .padding(.top, 10)
+        }
+        .frame(maxWidth: .infinity)
+        .frame(minHeight: geometry.size.height)
+        .padding()
+      }
+      .background(
+        Color(.braveGroupedBackground)
+          .edgesIgnoringSafeArea(.all)
+      )
+    }
+  }
+  
+  @ViewBuilder private var confirmedOrFailedTxView: some View {
+    GeometryReader { geometry in
+      ScrollView(.vertical) {
+        VStack(spacing: 10) {
+          Image(confirmationStore.activeTxStatus == .confirmed ? "tx-confirmed" : "tx-failed", bundle: .module)
+          Text(confirmationStore.activeTxStatus == .confirmed ? Strings.Wallet.confirmedTransactionTitle : Strings.Wallet.failedTransactionTitle)
+            .font(.title3.bold())
+            .foregroundColor(confirmationStore.activeTxStatus == .confirmed ? Color(.braveLabel) : Color(.braveErrorLabel))
+            .multilineTextAlignment(.center)
+            .padding(.top, 10)
+          Text(confirmationStore.activeTxStatus == .confirmed ? Strings.Wallet.confirmedTransactionDescription : Strings.Wallet.failedTransactionDescription)
+            .font(.subheadline)
+            .foregroundColor(Color(.secondaryBraveLabel))
+            .multilineTextAlignment(.center)
+          if confirmationStore.activeTxStatus == .error, let txProviderError = confirmationStore.transactionProviderErrorRegistry[confirmationStore.activeTransactionId] {
+            StaticTextView(text: "\(txProviderError.code): \(txProviderError.message)")
+              .frame(maxWidth: .infinity)
+              .frame(height: 100)
+              .background(Color(.tertiaryBraveGroupedBackground))
+              .clipShape(RoundedRectangle(cornerRadius: 5, style: .continuous))
+              .padding()
+              .background(
+                Color(.secondaryBraveGroupedBackground)
+              )
+              .clipShape(RoundedRectangle(cornerRadius: 10, style: .continuous))
+              .padding(.top, 10)
+          }
+          HStack {
+            if confirmationStore.activeTxStatus == .confirmed {
+              Button {
+                transactionDetails = confirmationStore.activeTxDetailsStore()
+              } label: {
+                Text(Strings.Wallet.confirmedTransactionReceiptButtonTitle)
+              }
+              .buttonStyle(BraveOutlineButtonStyle(size: .large))
+            }
+            Button {
+              onDismiss()
+            } label: {
+              Text(Strings.Wallet.confirmedTransactionCloseButtonTitle)
+            }
+            .buttonStyle(BraveFilledButtonStyle(size: .large))
```

Comment:
You can move this `buttonStyle` (and the one above) to the `HStack`

Reply:
they are different button styles tho

Reply:
updated in [02e39dd](https://github.com/brave/brave-ios/pull/6960/commits/02e39dd3500416bf27c899accce6252b59e78d3f)

Reply:
Oh sorry missed that!